README.md

```diff
@@ -2,3 +2,58 @@ Scalanus
 ========
 
 Yet another scripting language. Basically [Janus](https://github.com/mkaput/janus) done better, faster and funnier. Now in Scala. Much scalable. Very advanced. Such wow. Project for Scala Course at AGH UST.
+
+## TODO
+
+| | |
+|---|---|
+| 💔 | not implemented |
+| ⚙️ | in progress |
+| 🎉 | ready to go |
+
+### Language features
+
+| Feature | Grammar | IR  | Interpreter |
```

Comment:
Plan jest taki żeby zrobić 3 warstwy:
 - gramatykę która tworzy AST
 - reprezentację pośrednią, która jest uproszczonym AST po desugaringu i może jakiś optymalizacjach (do rozważenia to jeszcze jest)
 - interpreter to wykonywanie kodu